x-pack/platform/plugins/shared/fleet/server/types/models/package_policy.ts

```diff
@@ -435,20 +435,24 @@ export const OrphanedPackagePoliciesResponseSchema = schema.object({
   total: schema.number(),
 });
 
-export const DryRunPackagePolicySchema = schema.object({
-  ...PackagePolicyBaseSchema,
-  id: schema.maybe(schema.string()),
-  force: schema.maybe(schema.boolean()),
-  errors: schema.maybe(
-    schema.arrayOf(
-      schema.object({
-        message: schema.string(),
-        key: schema.maybe(schema.string()),
-      })
-    )
-  ),
-  missingVars: schema.maybe(schema.arrayOf(schema.string())),
-});
+export const DryRunPackagePolicySchema = PackagePolicySchema.extends(
```

Comment:
Had to fix this as it seems it was throwing invalid output 
<img width="1086" alt="Screenshot 2025-04-30 at 9 27 53 AM" src="https://github.com/user-attachments/assets/7c18bc86-2b9e-466b-80c4-4193b0036570" />